drools-wb-screens/drools-wb-scorecard-xls-editor/drools-wb-scorecard-xls-editor-client/src/main/java/org/drools/workbench/screens/scorecardxls/client/editor/URLHelper.java

```diff
@@ -27,14 +27,16 @@
 
     private static final String BASE = "scorecardxls/file";
 
-    public static String getServletUrl() {
-        return GWT.getModuleBaseURL() + BASE;
+    public static String getServletUrl( final String clientId ) {
+        return getServletUrl() + "?clientId=" + clientId;
     }
 
-    public static String getDownloadUrl( final Path path ) {
-        final StringBuilder sb = new StringBuilder( URLHelper.getServletUrl() );
-        sb.append( "?" ).append( FileManagerFields.FORM_FIELD_PATH ).append( "=" ).append( URL.encode( path.toURI() ) );
-        return sb.toString();
+    public static String getDownloadUrl( final Path path,
+                                         final String clientId ) {
+        return URLHelper.getServletUrl( clientId ) + "&" + FileManagerFields.FORM_FIELD_PATH + "=" + URL.encode( path.toURI() );
```

Comment:
You've correctly replace ```?``` with ```&``` here (see my comment in [here](https://github.com/droolsjbpm/drools-wb/pull/293/files#diff-d0226282fb580ad531c483c810f27435R36))